saleor/static/dashboard-next/components/AppLayout/AppLayout.tsx

```diff
@@ -202,6 +258,10 @@ const AppLayout = withStyles(styles, {
       const { isDark, toggleTheme } = useTheme();
       const [isDrawerOpened, setDrawerState] = React.useState(false);
       const [isMenuOpened, setMenuState] = React.useState(false);
+      const [menuToggle, setMenuToggle] = React.useState(
+        localStorage.getItem("menuToggle") === "true" ? true : false
```

Comment:
Please don't call data providers in pure components. Pass this as a prop instead.